server/__init__.py

```diff
@@ -169,13 +169,17 @@ def register_routes_sustainability(app):
       )
 
 
-def register_routes_datagemma(app, cfg):
+def register_routes_experiments(app, cfg):
   # Install blueprint for DataGemma page
   from server.routes.dev_datagemma import api as dev_datagemma_api
   app.register_blueprint(dev_datagemma_api.bp)
   from server.routes.dev_datagemma import html as dev_datagemma_html
   app.register_blueprint(dev_datagemma_html.bp)
 
+  # Install blueprint for Biomed NL experiment
+  from server.routes.biomed_nl import api as bio_api
```

Comment:
maybe we can make the folder dev_biomed_nl & call it dev_bio_api? Just to be consistent with the naming so when we go to change everything to "experiment", it'll be easier to find which things to change

Reply:
i was thinking we would put everything in an `experiments` folder, so it's all clearly labeled as an experiment. it should make future experiments easier to set up too.

/cc @gmechali @dwnoble

Reply:
I changed the file path to experiments/biomed_nl_api and the route to /api/experiments/biomed_nl/query.
Does this look okay? I can write up a quick doc for experiment routing to share out too if that helps.

Reply:
hm, maybe we can move this file to experiments/biomed_nl/api.py, so next we can add experiments/biomed_nl/html.py & then we can also have experiments/datagemma/api.py and experiments/datagemma/html.py